install-tests/react-native/patches/ccache-Podfile.patch

```diff
@@ -0,0 +1,18 @@
+--- app/ios/Podfile
++++ app/ios/Podfile
+@@ -39,5 +39,15 @@
+       :mac_catalyst_enabled => false
+     )
+     __apply_Xcode_12_5_M1_post_install_workaround(installer)
++    # Using ccache when building
++    installer.pods_project.targets.each do |target|
++      target.build_configurations.each do |config|
++        # Using the un-qualified names means you can swap in different implementations, for example ccache
```

Comment:
Where are we performing this swap?  Shouldn't there be a step somewhere to replace `clang` and `clang++` with a symbolic link to `ccache`?

Reply:
According to https://reactnative.dev/docs/build-speed#xcode-specific-setup this is enough to have XCode pick up the ccache executable if it's in the `PATH` and [that happens here](https://github.com/realm/realm-js/blob/kh/install-test-refactored/.github/workflows/install-test-react-native.yml#L62).